.evergreen/config.yml

```diff
@@ -1,48 +1,191 @@
+# Mark a failure as a system/bootstrap failure (purple box) rather then a task
```

Comment:
well, Jeff has explained the rationale behind the renaming, right? I am ok with either. I think everything has pros and cons?

Reply:
I do not recall Jeff explaining this. If you could link to a comment, this would be helpful. If this was a verbal explanation, perhaps you could repeat it here.

Reply:
https://github.com/mongodb/mongo-hibernate/pull/9#discussion_r1847208311

Reply:
We decided to go with `config.yml`, to align with other teams.